pl_bolts/models/rl/dqn_model.py

```diff
@@ -171,7 +171,7 @@ def run_n_episodes(self, env, n_epsiodes: int = 1, epsilon: float = 1.0) -> List
             while not done:
                 self.agent.epsilon = epsilon
                 action = self.agent(episode_state, self.device)
-                next_state, reward, done, _ = self.env.step(action[0])
+                next_state, reward, done, _ = env.step(action[0])
```

Comment:
shall we also assign the env back, `self.env = env`?

Reply:
if we assign the test env to `self.env` in `test_step`, when training is done after testing, it'll use the test env (without random seed) instead of what it was initialized with for training (env without random seed)